cargo-dist/src/manifest.rs

```diff
@@ -124,13 +124,6 @@ pub(crate) fn load_and_merge_manifests(
             output.ci = Some(val);
         };
 
-        // Just merge all the system-specific info
```

Comment:
Mind leaving this comment in, attached to the `extend` calls below?

Reply:
oh sure, i wasn't sure if "system-specific info" applied to all 3 calls or now, i'll update that now